runtime/picasso/src/lib.rs

```diff
@@ -773,6 +773,43 @@ impl assets::Config for Runtime {
 	type GovernanceRegistry = GovernanceRegistry;
 }
 
+#[cfg(feature = "develop")]
+parameter_types! {
+	pub const MaxVestingSchedule: u32 = 2;
+	pub const MinVestedTransfer: u64 = 1_000_000;
+}
+
+#[cfg(feature = "develop")]
+impl vesting::Config for Runtime {
+	type Currency = Tokens;
+	type Event = Event;
+	type MaxVestingSchedules = MaxVestingSchedule;
+	type MinVestedTransfer = MinVestedTransfer;
+	type VestedTransferOrigin = system::EnsureSigned<AccountId>;
+	type WeightInfo = ();
+}
```

Comment:
The `bonded-finance` pallet needs the `vesting` pallet, thus its runtime addition